### PR TITLE
updating the RUM home page

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?
Updates the new RUM home page to our current home page style

### Motivation
Making RUM functionality more clear.

### Preview

https://docs-staging.datadoghq.com/kaylyn/rum-home/real_user_monitoring/

### CODE REVIEW

File: content/en/real_user_monitoring/_index.md [hunk 1]
```diff
@@ -26,15 +26,31 @@ Datadog's Real User Monitoring (RUM) gives you end-to-end visibility into the re
...
+
+{{< partial name="rum/rum-getting-started.html" >}}
+</br>
+## Explore Datadog RUM
+
+### Out of the Box Dashboards
+
+Discover all data collected automatically and displayed on out of the box dashboards.
+
+{{< img src="real_user_monitoring/dashboards/rum_applications.gif" alt="RUM applications" >}}
```
Comment (hdelaby) [inline content/en/real_user_monitoring/_index.md:41]: This GIF's resolution is not so great, can we record it better? Seems like I'm definitely not a GIF master 🥉

File: content/en/real_user_monitoring/_index.md [hunk 2]
```diff
@@ -26,15 +26,31 @@ Datadog's Real User Monitoring (RUM) gives you end-to-end visibility into the re
...
-  {{< nextlink href="/real_user_monitoring/explorer/">}}<u>RUM Search</u>: Search through your page views.{{< /nextlink >}}
-  {{< nextlink href="/real_user_monitoring/explorer/analytics">}}<u>RUM Analytics</u>: Get insights from all your events.{{< /nextlink >}}
-{{< /whatsnext >}}
+## Getting started
+
+Select your application type to start collecting RUM data:
+
+{{< partial name="rum/rum-getting-started.html" >}}
+</br>
+## Explore Datadog RUM
```
Comment (hdelaby) [inline content/en/real_user_monitoring/_index.md:35]: We're missing a sentence below. Something like APM: "Now that you’ve configured your application to send RUM data to Datadog, start getting insights into your application performance:"

File: content/en/real_user_monitoring/_index.md [hunk 3]
```diff
@@ -26,15 +26,31 @@ Datadog's Real User Monitoring (RUM) gives you end-to-end visibility into the re
...
-{{< /whatsnext >}}
-{{< whatsnext desc="Explore your RUM events:">}}
-  {{< nextlink href="/real_user_monitoring/explorer/">}}<u>RUM Search</u>: Search through your page views.{{< /nextlink >}}
-  {{< nextlink href="/real_user_monitoring/explorer/analytics">}}<u>RUM Analytics</u>: Get insights from all your events.{{< /nextlink >}}
-{{< /whatsnext >}}
+## Getting started
+
+Select your application type to start collecting RUM data:
+
+{{< partial name="rum/rum-getting-started.html" >}}
```
Comment (hdelaby) [inline content/en/real_user_monitoring/_index.md:33]: The icons look bad on my MBP screen (resolution?): 
image

**Review by hdelaby — COMMENTED** [review]
Could we also link to the mobile blog post in the "further reading" section? https://www.datadoghq.com/blog/datadog-mobile-rum/

Thanks for this PR! 👍

**Comment by apigirl** [comment]
closing this PR in favor of https://github.com/DataDog/documentation/pull/9548

**Pull request closed without merging**